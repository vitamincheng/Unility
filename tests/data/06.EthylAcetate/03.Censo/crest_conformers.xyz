14
 Energy =        -307.65065846 Eh        #Cluster:     1
  C     -2.0659812777     -0.9790245736     -0.0104103689    
  C     -0.9961370428      0.0739462490      0.0132068871    
  H     -2.9816021719     -0.5797190438      0.4277162415    
  H     -1.7493587786     -1.8848184298      0.5101214620    
  H     -2.2676630140     -1.2399057968     -1.0557379490    
  O     -1.1977036739      1.2722275583     -0.0003156325    
  O      0.2348926528     -0.4716985032      0.0219601186    
  C      1.3538689278      0.4625669355     -0.0122981961    
  C      2.6219012458     -0.3573070571     -0.0043093441    
  H      1.2843414427      1.1201337123      0.8608938161    
  H      1.2664025245      1.0770620326     -0.9147873578    
  H      2.6864038176     -0.9709313733      0.8992309101    
  H      3.4857853405      0.3140934739     -0.0270827740    
  H      2.6715566426     -1.0122722902     -0.8793513361    
14
 Energy =        -307.65017457 Eh        #Cluster:     2
  C     -2.0711701353     -0.8729049682      0.1513153965    
  C     -0.8840449254      0.0343870073      0.0046029790    
  H     -2.1177280592     -1.6043548454     -0.6580213425    
  H     -2.9845289129     -0.2782498339      0.1900950556    
  H     -1.9705930027     -1.4204960866      1.0955519548    
  O     -0.8567536863      1.2029089402      0.3357787237    
  O      0.1687171908     -0.6204063082     -0.5258533211    
  C      1.4144338376      0.1283269722     -0.6503578516    
  C      2.1858416426      0.1181168964      0.6524835371    
  H      1.1830706492      1.1461018636     -0.9774995735    
  H      1.9588290199     -0.3927278506     -1.4418454688    
  H      1.6342951267      0.6387705074      1.4402231847    
  H      3.1445051390      0.6276896228      0.5095214272    
  H      2.3869871608     -0.9077847064      0.9758261034    
