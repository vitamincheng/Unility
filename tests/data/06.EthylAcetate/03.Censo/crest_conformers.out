
         ______________________________________________________________
        |                                                              |
        |                                                              |
        |                   CENSO - Commandline ENSO                   |
        |                           v 1.2.0                            |
        |    energetic sorting of CREST Conformer Rotamer Ensembles    |
        |                    University of Bonn, MCTC                  |
        |                           Feb 2021                           |
        |                 based on ENSO version 2.0.1                  |
        |                     F. Bohle and S. Grimme                   |
        |                                                              |
        |______________________________________________________________|

        Please cite: 
        S. Grimme, F. Bohle, A. Hansen, P. Pracht, S. Spicher, and M. Stahn 
        J. Phys. Chem. A 2021, 125, 19, 4039-4054.
        DOI: https://doi.org/10.1021/acs.jpca.1c00971
        
        This program is distributed in the hope that it will be useful,
        but WITHOUT ANY WARRANTY; without even the implied warranty of
        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.


----------------------------------------------------------------------------------------------------
                                             PARAMETERS                                             
----------------------------------------------------------------------------------------------------

program call: /home/vitamin/.local/bin/censo -inp crest_conformers.xyz
The configuration file .censorc is read from /home/vitamin/Simulation/00.Solvent/21.Ethyl Acetate (6.0.1)/03.Censo/.censorc.
Reading conformer rotamer ensemble from: /home/vitamin/Simulation/00.Solvent/21.Ethyl Acetate (6.0.1)/03.Censo/crest_conformers.xyz.
Reading file: censo_solvents.json

Reading file: censo_orca_editable.dat


--------------------------------------------------
               CRE SORTING SETTINGS               
--------------------------------------------------

number of atoms in system:                                     14
number of considered conformers:                               2
number of all conformers from input:                           2
charge:                                                        0
unpaired:                                                      0
solvent:                                                       chcl3
temperature:                                                   298.15
calculate mRRHO contribution:                                  on
consider symmetry for mRRHO contribution:                      on
cautious checking for error and failed calculations:           on
checking the DFT-ensemble using CREST:                         off
maxthreads:                                                    2
omp:                                                           4
automatically balance maxthreads and omp:                      on

--------------------------------------------------
          CRE CHEAP-PRESCREENING - PART0          
--------------------------------------------------
part0:                                                         on
starting number of considered conformers:                      2
program for part0:                                             orca
functional for fast single-point:                              r2scan-3c
basis set for fast single-point:                               def2-mTZVPP
threshold g_thr(0) for sorting in part0:                       40.0
Solvent model used with xTB:                                   alpb

short-notation:
r2scan-3c // GFNn-xTB (Input geometry)

--------------------------------------------------
             CRE PRESCREENING - PART1             
--------------------------------------------------
part1:                                                         on
program for part1:                                             orca
functional for initial evaluation:                             r2scan-3c
basis set for initial evaluation:                              def2-mTZVPP
calculate mRRHO contribution:                                  on
program for mRRHO contribution:                                xtb
GFN version for mRRHO and/or GBSA_Gsolv:                       gfn2
Apply constraint to input geometry during mRRHO calculation:   on
solvent model applied with xTB:                                alpb
evaluate at different temperatures:                            off
threshold g_thr(1) and G_thr(1) for sorting in part1:          60.0
solvent model for Gsolv contribution of part1:                 smd

short-notation:
r2scan-3c + SMD[chcl3] + GmRRHO(GFN2[alpb]-bhess) // GFNn-xTB (Input geometry)

--------------------------------------------------
                 NMR MODE SETTINGS                
--------------------------------------------------
part4:                                                         on
calculate couplings (J):                                       on
program for coupling calculations:                             orca
solvation model for coupling calculations:                     cpcm
functional for coupling calculation:                           pbe0-d4
basis set for coupling calculation:                            pcJ-0

calculate shieldings (S):                                      on
program for shielding calculations:                            orca
solvation model for shielding calculations:                    cpcm
functional for shielding calculation:                          revtpss-novdw
basis set for shielding calculation:                           pcSseg-1

Calculating proton spectrum:                                   on
reference for 1H:                                              TMS
resonance frequency:                                           500.0
END of parameters


------------------------------------------------------------
               PATHS of external QM programs                
------------------------------------------------------------

The following program paths are used:
    ORCA:         /home/vitamin/orca_6_0_1_linux_x86-64_avx2_shared_openmpi416
    ORCA Version: 6.01
    xTB:          /home/vitamin/xtb/bin/xtb


----------------------------------------------------------------------------------------------------
                            Processing data from previous run (enso.json)                           
----------------------------------------------------------------------------------------------------

INFORMATION: No restart information exists and is created during this run!


----------------------------------------------------------------------------------------------------
                                   CRE CHEAP-PRESCREENING - PART0                                   
----------------------------------------------------------------------------------------------------

program:                                                       orca
functional for part0:                                          r2scan-3c
basis set for part0:                                           def2-mTZVPP
threshold g_thr(0):                                            40.0
starting number of considered conformers:                      2
temperature:                                                   298.15

Calculating efficient gas-phase single-point energies:
The efficient gas-phase single-point is calculated for:
CONF1, CONF2

Constructed folders!

Starting 2 ALPB-Gsolv calculations
Running single-point in CONF1/part0_sp
Running single-point in CONF2/part0_sp
Running ALPB_GSOLV calculation in 03.Censo/CONF1/part0_sp
Running ALPB_GSOLV calculation in 03.Censo/CONF2/part0_sp
Tasks completed!

The efficient gas-phase single-point was successful for CONF1/part0_sp: E(DFT) = -307.64369813 Gsolv = -0.01433678
The efficient gas-phase single-point was successful for CONF2/part0_sp: E(DFT) = -307.64363805 Gsolv = -0.01395540

----------------------------------------------------------------------------------------------------
                   Removing high lying conformers by improved energy description                    
----------------------------------------------------------------------------------------------------

CONF#       E [Eh] ΔE [kcal/mol]       E [Eh]   Gsolv [Eh]         gtot    ΔE(DFT)     ΔGsolv      Δgtot
          GFN2-xTB      GFN2-xTB    r2scan-3c         alpb         [Eh] [kcal/mol] [kcal/mol] [kcal/mol]
            [alpb]        [alpb]                    [gfn2]                                              
CONF1  -20.8002595          0.26 -307.6436981   -0.0143368 -307.6580349       0.00       0.00       0.00     <------
CONF2  -20.8006665          0.00 -307.6436381   -0.0139554 -307.6575935       0.04       0.24       0.28
----------------------------------------------------------------------------------------------------

Number of conformers observed within the following Δg windows:
Δg [kcal/mol]  #CONF   sum(Boltzmann_weights)
---------------------------------------------
   0 - 0.5        2          1.00
---------------------------------------------

All relative (free) energies are below the initial g_thr(0) threshold of 40.0 kcal/mol.
All conformers are considered further.

Calculating Boltzmann averaged (free) energy of ensemble on input geometries (not DFT optimized)!

temperature /K:   avE(T) /a.u.   avG(T) /a.u. 
----------------------------------------------------------------------------------------------------
    298.15        -307.6436750    -307.6578649     <<==part0==
----------------------------------------------------------------------------------------------------


>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>END of Part0<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
Ran part0 in 3.5156 seconds

----------------------------------------------------------------------------------------------------
                                      CRE PRESCREENING - PART1                                      
----------------------------------------------------------------------------------------------------

program:                                                       orca
functional for part1 and 2:                                    r2scan-3c
basis set for part1 and 2:                                     def2-mTZVPP
Solvent:                                                       chcl3
solvent model for Gsolv contribution:                          smd
threshold g_thr(1) and G_thr(1):                               60.0
starting number of considered conformers:                      2
calculate mRRHO contribution:                                  on
program for mRRHO contribution:                                xtb
GFN version for mRRHO and/or GBSA_Gsolv:                       gfn2
Apply constraint to input geometry during mRRHO calculation:   on
temperature:                                                   298.15

Calculating single-point energies and solvation contribution (G_solv):
The prescreening_single-point is calculated for:
CONF1, CONF2

Constructed folders!
Running single-point in CONF1/r2scan-3c
Running single-point in CONF2/r2scan-3c
Tasks completed!

prescreening_single-point calculation was successful for CONF1/r2scan-3c: -307.65243107
prescreening_single-point calculation was successful for CONF2/r2scan-3c: -307.65181582

--------------------------------------------------
          Removing high lying conformers          
--------------------------------------------------

CONF#  E(GFNn-xTB) ΔE(GFNn-xTB)       E [Eh]   Gsolv [Eh]         gtot      Δgtot
            [a.u.]   [kcal/mol]    r2scan-3c   incl. in E         [Eh] [kcal/mol]
                                       [SMD]                                     
CONF1 -307.6506585         0.00 -307.6524311    0.0000000 -307.6524311       0.00     <------
CONF2 -307.6501746         0.30 -307.6518158    0.0000000 -307.6518158       0.39

All relative (free) energies are below the g_thr(1) threshold of 60.0 kcal/mol.
All conformers are considered further.
--------------------------------------------------

Calculating prescreening G_mRRHO with implicit solvation!
The prescreening G_mRRHO calculation is now performed for:
CONF1, CONF2

Constructed folders!

Starting 2 G_RRHO calculations.
Running GFN2-xTB mRRHO in CONF1/rrho_part1
Running GFN2-xTB mRRHO in CONF2/rrho_part1
Tasks completed!

The prescreening G_mRRHO run @ c1 was successful for CONF1/rrho_part1: 0.08243554 S_rot(sym)= 0.0000000 using= 0.0824355
The prescreening G_mRRHO run @ c1 was successful for CONF2/rrho_part1: 0.08289559 S_rot(sym)= 0.0000000 using= 0.0828956

--------------------------------------------------
         * Gibbs free energies of part1 *         
--------------------------------------------------

CONF#  G(GFNn-xTB) ΔG(GFNn-xTB)       E [Eh]   Gsolv [Eh]  GmRRHO [Eh]         Gtot      ΔGtot
            [a.u.]   [kcal/mol]    r2scan-3c   incl. in E         GFN2         [Eh] [kcal/mol]
                                       [SMD]              [alpb]-bhess                        
CONF1 -307.5682229         0.00 -307.6524311    0.0000000    0.0824355 -307.5699955       0.00     <------
CONF2 -307.5672790         0.59 -307.6518158    0.0000000    0.0828956 -307.5689202       0.67

Number of conformers observed within the following ΔG windows:
ΔG [kcal/mol]  #CONF   sum(Boltzmann_weights)
---------------------------------------------
   0 - 0.5        1          0.76
   0 - 1.0        2          1.00
---------------------------------------------

Additional global 'fuzzy-threshold' based on the standard deviation of (G_mRRHO):
Std_dev(G_mRRHO) = 0.204 kcal/mol
Fuzzythreshold   = 0.188 kcal/mol
Final sorting threshold G_thr(1) = 60.000 + 0.188 = 60.188 kcal/mol
Spearman correlation coefficient between (E + Solv) and (E + Solv + mRRHO) = 1.000

All relative (free) energies are below the initial G_thr(1) threshold of 60.0 kcal/mol.
All conformers are considered further.

Calculating Boltzmann averaged free energy of ensemble on input geometries (not DFT optimized)!

temperature /K:   avE(T) /a.u. avGmRRHO(T) /a.u. avGsolv(T) /a.u.   avG(T) /a.u.
----------------------------------------------------------------------------------------------------
    298.15        -307.6522819        0.0825471        0.0000000   -307.5697347      <<==part1==
----------------------------------------------------------------------------------------------------


Calculating unbiased GFNn-xTB energy
Constructed folders!

Starting 2 xTB - single-point calculations.
gfn2-xTB energy for CONF1/GFN_unbiased = -20.8002595
gfn2-xTB energy for CONF2/GFN_unbiased = -20.8006665
Tasks completed!


>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>END of Part1<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
Ran part1 in 5.3532 seconds

----------------------------------------------------------------------------------------------------
                                          NMR MODE - PART4                                          
----------------------------------------------------------------------------------------------------

calculate coupling constants:                                  on
prog4J - program for coupling constant calculation:            orca
funcJ  - functional for coupling constant calculation:         pbe0-d4
basisJ - basis for coupling constant calculation:              pcJ-0
sm4J - solvent model for the coupling calculation:             cpcm

calculate shielding constants σ:                               on
prog4S - program for shielding constant calculation:           orca
funcS - functional for shielding constant calculation:         revtpss-novdw
basisS - basis for shielding constant calculation:             pcSseg-1
sm4S - solvent model for the shielding calculation:            cpcm

Calculating proton spectrum:                                   on
reference for 1H:                                              TMS
spectrometer frequency:                                        500.0

INFORMATION: Conformers have not been optimized at DFT level!!!
             Use results with care!

Considering the following 2 conformers:
CONF1, CONF2


--------------------------------------------------
      * Gibbs free energies used in part4 *       
--------------------------------------------------

CONF#       E [Eh]   Gsolv [Eh]  GmRRHO [Eh]         Gtot      ΔGtot Boltzmannweight
         r2scan-3c   incl. in E         GFN2         [Eh] [kcal/mol]   % at 298.15 K
             [SMD]              [alpb]-bhess                                        
CONF1 -307.6524311    0.0000000    0.0824355 -307.5699955       0.00           75.75     <------
CONF2 -307.6518158    0.0000000    0.0828956 -307.5689202       0.67           24.25

Conformers that are below the Boltzmann-thr of 95.0:
CONF1, CONF2

Constructed folders!

Performing coupling constant calculations:

Starting 2 coupling constants calculations
Running coupling calculation in CONF1/NMR
Running coupling calculation in CONF2/NMR
Tasks completed!

Coupling constant calculation was successful for CONF1/NMR
Coupling constant calculation was successful for CONF2/NMR

Performing shielding constant calculations:

Starting 2 shielding constants calculations
Running shielding calculation in CONF1/NMR         
Running shielding calculation in CONF2/NMR         
Tasks completed!

Shielding constant calculation was successful for CONF1/NMR
Shielding constant calculation was successful for CONF2/NMR

Generating file anmr_enso for processing with the ANMR program.

Writing .anmrrc!
INFORMATION: The reference molecule shielding constant has been calucated with different settings.
             You can always adjust the reference in the file .anmrrc manually!
WARNING:     The geometry optimization of the reference molecule was calculated with SMD (sm2)!
WARNING:     The reference shielding constant was calculated with SMD (sm4_s)!
WARNING:     The reference shielding constant was calculated with the basis def2-TZVP (basisS) instead of pcSseg-1!
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
<censo_qm.cfg.NmrRef object at 0x75e57723580>
revtpss-novdw
r2scan-3c
pcSseg-1
chcl3
h {'atomic_number': 1, 'sigma': 0.0, 'tm': 'h_tm_shieldings', 'orca': 'h_orca_shieldings', 'adf': 'h_adf_shieldings', 'ref_mol': 'TMS', 'active': True}
c {'atomic_number': 6, 'sigma': 0.0, 'tm': 'c_tm_shieldings', 'orca': 'c_orca_shieldings', 'adf': 'c_adf_shieldings', 'ref_mol': 'TMS', 'active': False}
f {'atomic_number': 9, 'sigma': 0.0, 'tm': 'f_tm_shieldings', 'orca': 'f_orca_shieldings', 'adf': 'f_adf_shieldings', 'ref_mol': 'CFCl3', 'active': False}
si {'atomic_number': 14, 'sigma': 0.0, 'tm': 'si_tm_shieldings', 'orca': 'si_orca_shieldings', 'adf': 'si_adf_shieldings', 'ref_mol': 'TMS', 'active': False}
p {'atomic_number': 15, 'sigma': 0.0, 'tm': 'p_tm_shieldings', 'orca': 'p_orca_shieldings', 'adf': 'p_adf_shieldings', 'ref_mol': 'TMP', 'active': False}
h {'atomic_number': 1, 'sigma': '31.820', 'tm': 'h_tm_shieldings', 'orca': 'h_orca_shieldings', 'adf': 'h_adf_shieldings', 'ref_mol': 'TMS', 'active': True}
c {'atomic_number': 6, 'sigma': 0.0, 'tm': 'c_tm_shieldings', 'orca': 'c_orca_shieldings', 'adf': 'c_adf_shieldings', 'ref_mol': 'TMS', 'active': False}
f {'atomic_number': 9, 'sigma': 0.0, 'tm': 'f_tm_shieldings', 'orca': 'f_orca_shieldings', 'adf': 'f_adf_shieldings', 'ref_mol': 'CFCl3', 'active': False}
si {'atomic_number': 14, 'sigma': 0.0, 'tm': 'si_tm_shieldings', 'orca': 'si_orca_shieldings', 'adf': 'si_adf_shieldings', 'ref_mol': 'TMS', 'active': False}
p {'atomic_number': 15, 'sigma': 0.0, 'tm': 'p_tm_shieldings', 'orca': 'p_orca_shieldings', 'adf': 'p_adf_shieldings', 'ref_mol': 'TMP', 'active': False}

Generating plain nmrprop.dat files for each populated conformer.
These files contain all calculated shielding and coupling constants.
The files can be read by ANMR using the keyword '-plain'.

Tasks completed!


Averaged shielding constants:
# in coord  element  σ(sigma)  SD(σ based on SD Gsolv)  SD(σ by 0.4 kcal/mol)       shift        σ_ref
---------------------------------------------------------------------------------------------------------
 3             h       29.69          0.000000                 0.001323            2.13        31.820
 4             h       29.69          0.000000                 0.001323            2.13        31.820
 5             h       29.69          0.000000                 0.001323            2.13        31.820
 10            h       27.70          0.000000                 0.041269            4.12        31.820
 11            h       27.70          0.000000                 0.041269            4.12        31.820
 12            h       30.50          0.000000                 0.020565            1.32        31.820
 13            h       30.50          0.000000                 0.020565            1.32        31.820
 14            h       30.50          0.000000                 0.020565            1.32        31.820
---------------------------------------------------------------------------------------------------------

# in coord  element  σ(sigma)  min(σ)* CONFX   max(σ)* CONFX  Δ(max-min)
---------------------------------------------------------------------------------------------------------
 3             h       29.69    29.68 CONF2     29.69 CONF1      0.01
 4             h       29.69    29.68 CONF2     29.69 CONF1      0.01
 5             h       29.69    29.68 CONF2     29.69 CONF1      0.01
 10            h       27.70    27.52 CONF2     27.76 CONF1      0.25
 11            h       27.70    27.52 CONF2     27.76 CONF1      0.25
 12            h       30.50    30.47 CONF1     30.59 CONF2      0.12
 13            h       30.50    30.47 CONF1     30.59 CONF2      0.12
 14            h       30.50    30.47 CONF1     30.59 CONF2      0.12
---------------------------------------------------------------------------------------------------------
* min(σ) and max(σ) are averaged over the chemical equivalent atoms, but not Boltzmann weighted.

***---------------------------------------------------------***
Printing most relevant errors again, just for user convenience:
INFORMATION: Conformers have not been optimized at DFT level!!!
             Use results with care!

***---------------------------------------------------------***

>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>END of Part4<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
Ran part4 in 10.3626 seconds


Part                : #conf       time      time (including restarts)
-----------------------------------------------------------------------
Input               :     2        -            -    
Part0_all           :     2       3.52 s     3.52 s
Part1_initial_sort  :     2       4.34 s     4.34 s
Part1_all           :     2       5.35 s     5.35 s
Part4               :     2      10.36 s    10.36 s
-----------------------------------------------------------------------
All parts           :     -      19.23 s    19.23 s

CENSO all done!
